composer.json

```
@@ -91,6 +91,13 @@
             "Frontend\\User\\": "src/User/src/"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "FrontendTest\\Common\\": "test/Common",
+            "FrontendTest\\Functional\\": "test/Functional/",
+            "FrontendTest\\Unit\\": "test/Unit"
+        }
+    },
     "scripts": {
         "post-create-project-cmd": [
```

```suggestion
        "static-analysis": "psalm --shepherd --stats",
        "post-create-project-cmd": [
```
Maybe move it lower, after `serve`.